app/views/users/show.html.erb

```diff
@@ -1 +1,56 @@
-<div>I should show user info for one user. Probably in some dl, (dt/dd)* type list. Please fix me.</div>
+<h1> <%= @user.name %> </h1>
+
+<dl class="row mt-5">
+  <dt class="col-sm-2 text-md-right">Email address:</dt>
+  <dd class="col-sm-10"><%= @user.email %><br></dd>
+  <dt class="col-sm-2 text-md-right">Phone number:</dt>
+  <dd class="col-sm-10"><%= @user.phone_number %><br></dd>
+  <dt class="col-sm-2 text-md-right">Joined on:</dt>
+  <dd class="col-sm-10"><%= @user.created_at %><br></dd>
+  <dt class="col-sm-2 text-md-right">Role:</dt>
+  <dd class="col-sm-10"><%= @user.role.humanize %><br></dd>
+  <dt class="col-sm-2 text-md-right">Status:</dt>
+  <dd class="col-sm-10">
+    <% if @user.deactivated? %>
+      Inactive
+    <% else %>
+      Active
+    <% end %></dd>
+</dl>
+
+<dt class="col-sm-2 text-md-right"><%= link_to 'Edit User Details', edit_user_path(@user) %></dt>
+<br><br>
+
+<h2>Loan history</h2>
+<% if @loans %>
```

Comment:
This will not do what you intend. It will print the table even if the collection is empty (because it will not be `nil`). You can use `@loans.empty?` instead.

Reply:
@mark100net we have been doing a "squash and merge" on the PR so that will be taken care of when I merge it in.